frontend/src/styles/PlayerAnswerButton.module.scss

```diff
@@ -0,0 +1,144 @@
+.answerButtonBlueStar {
+    position: absolute;
+    width: 500px;
+    height: 300px;
+    left: 745px;
+    top: 125px;
+    background: #01FFF4;
+    border-radius: 17px;
+    color:  #27163F;
+    border-width: 0px;
+}
+
+.answerButtonPinkCircle {
+    position: absolute;
+    width: 500px;
+    height: 300px;
```

Comment:
There's a lot of repetition of these 3 lines of code in your answer buttons. Could you make them into another class so you don't repeat these 3 lines on every answer button? :)